package.json

```diff
@@ -3,6 +3,33 @@
   "title": "jQuery",
   "description": "JavaScript library for DOM operations",
   "version": "4.0.0-pre",
+  "type": "commonjs",
```

Comment:
There are sites that look for `"type": "module"` as a way of detecting if a package is written in ESM; yes they can inspect `"exports"` too but looking only at `type` is a lot simpler. If possible I would suggest you make this `"type": "module"`, and add a `package.json` with `"type": "commonjs"` in a subfolder if needed. (And any CommonJS files at the root would need to get `.cjs` extensions.)